Update live docs for NLB-SG feature release (#3332)
kubernetes-sigs/aws-load-balancer-controller: docs/guide/service/annotations.md

        ```
        service.beta.kubernetes.io/aws-load-balancer-internal: "true"
        ```

> - <a name="security-groups">`service.beta.kubernetes.io/aws-load-balancer-security-groups`</a>  specifies the securityGroups you want to attach to NLB.

Reviewer: nit: may be "specifies the frontend securityGroups you want to attach to NLB"

> - <a name="security-groups">`service.beta.kubernetes.io/aws-load-balancer-security-groups`</a>  specifies the frontend securityGroups you want to attach to NLB.

Reviewer 2: ```suggestion
- <a name="security-groups">`service.beta.kubernetes.io/aws-load-balancer-security-groups`</a>  specifies the frontend securityGroups you want to attach to an NLB.
```

    !!!note ""

>         When this annotation is not present, the controller will automatically create one security group, the security group will be attached to the LoadBalancer and allow access from `inbound-cidrs` to the `listen-ports`.

Reviewer 2: ```suggestion
        When this annotation is not present, the controller will automatically create one security group. The security group will be attached to the LoadBalancer and allow access from `inbound-cidrs` to the `listen-ports`.
```

>         Also, the securityGroups for instance/ENI will be modified to allow inbound traffic from this securityGroup.

Reviewer 2: ```suggestion
        Also, the securityGroups for target instances/ENIs will be modified to allow inbound traffic from this securityGroup.
```

    !!!note ""

>         If you specify this annotation, you need to configure the security groups on your instance/ENI to allow inbound traffic from the load balancer. You could also set the [`manage-backend-security-group-rules`](#manage-backend-sg-rules) if you want the controller to manage the access rules.

Reviewer 2: ```suggestion
        If you specify this annotation, you need to configure the security groups on your target instances/ENIs to allow inbound traffic from the load balancer. You could also set the [`manage-backend-security-group-rules`](#manage-backend-sg-rules) if you want the controller to manage the security group rules.
```

    !!!tip ""

>         Both name or ID of securityGroups are supported. Name matches a `Name` tag, not the `groupName` attribute.

Reviewer 2: ```suggestion
        Both name and ID of securityGroups are supported. Name matches a `Name` tag, not the `groupName` attribute.
```
